src/main.scss

```
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  font-family: Arial;
```

На Linux нет Arial, что тогда загрузится?
```suggestion
  font-family: Arial, sans-serif;
```